packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/index.web.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
@hxlaf Please export here `./generated/openapi/typescript-axios/index.ts` as a wildcard (everything from it - see other packages for examples.

Reply:
Thank you! Done.